src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -251,6 +251,90 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
     }
   }
 
+  test("Verify refresh-incremental (append-only) throws exception if no new files found.") {
+    withTempDir { testDir =>
+      val refreshTestLocation = testDir + "/refresh"
+      withSQLConf(IndexConstants.REFRESH_APPEND_ENABLED -> "true") {
+        // Setup. Create sample data and index.
+        FileUtils.delete(new Path(refreshTestLocation))
```

Comment:
No need to do this now that we are using `withTemp*`.

Reply:
thanks @imback82 , fixed